src/state/queries/post.ts

```diff
@@ -111,22 +113,32 @@ export function usePostLikeMutationQueue(
   return [queueLike, queueUnlike]
 }
 
-function usePostLikeMutation() {
+function usePostLikeMutation(
+  logContext: 'FeedItem' | 'PostThreadItem' | 'Post',
+) {
   return useMutation<
     {uri: string}, // responds with the uri of the like
     Error,
     {uri: string; cid: string} // the post's uri and cid
   >({
-    mutationFn: post => getAgent().like(post.uri, post.cid),
+    mutationFn: post => {
+      logEvent('post:like', {logContext})
+      return getAgent().like(post.uri, post.cid)
+    },
     onSuccess() {
       track('Post:Like')
     },
   })
 }
 
-function usePostUnlikeMutation() {
+function usePostUnlikeMutation(
+  logContext: 'FeedItem' | 'PostThreadItem' | 'Post',
```

Comment:
Oh ya know, should we re-use the `Events['post:unlike']['logContext']` type for these to keep in sync?

Reply:
yeah i was leaning towards that as well.. ok lemme do that real quick

Reply:
ok yeah i did this for mutations, but kept explicit enumerations for components since those are more local and i think you might only want a subset for props.